### PR TITLE
Addresses #3096 NTR pulmonary venule

### PR DESCRIPTION
**Pull request by aleixpuigb**
Addresses #3096 NTR pulmonary venule

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224613,6 +224613,17 @@ relationship: overlaps UBERON:0018227 ! pulmonary lymphatic vessel
 relationship: surrounds UBERON:0010369 ! secondary pulmonary lobule
 property_value: dcterms-date "2023-10-23T08:52:12Z" xsd:dateTime
 
+[Term]
+id: UBERON:8600024
+name: pulmonary venule
+def: "A venule that is part of a lung and drains oxygenated blood from pulmonary capillaries. A pulmonary venule merges into a pulmonary vein branch that run in the pulmonary interlobular septa." [PMID:22283705, PMID:30247823]
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:224619]:
```suggestion
def: "A venule that is part of a lung and drains oxygenated blood from pulmonary capillaries. A pulmonary venule merges into a pulmonary vein branch that runs in the pulmonary interlobular septa." [PMID:22283705, PMID:30247823]
```

This sounds like a "pulmonary vein branch", not a pulmonary venule, passes through a pulmonary interlobular septum. Is that aligned with the comment [here](https://github.com/obophenotype/uberon/pull/3095#discussion_r1369754009)?

**Review by bvarner-ebi — CHANGES_REQUESTED** [review]
See inline comment.
Also, with the current modelling, this class does not seem to be related to 'pulmonary venous system'. Consider if it should be.

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224619]: You are correct, I have to edit interlobular septa accordingly.

Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224619]: A new ticket #3115 has been opened to correct this.

**Comment by aleixpuigb** [comment]
> See inline comment. Also, with the current modelling, this class does not seem to be related to 'pulmonary venous system'. Consider if it should be.

Thank you for your suggestion, I have added 'part of' some 'pulmonary venous system'.

**Review by bvarner-ebi — APPROVED** [review]
No breaking changes found.
Defer content review to SME. After a cursory online review, there does not seem to be consensus on pulmonary venous vasculature terminology.

**Pull request merged**